marklogic-data-hub-central/ui/src/App.tsx

```diff
@@ -77,6 +77,16 @@ const App: React.FC<Props> = ({history, location}) => {
 
   }, [user]);
 
+  useEffect(() => {
+    // On route change...
+    axios.get('/api/environment/project-info')
```

Comment:
This was '/api/environment/project' earlier. Should this be project-info now?

Reply:
yeah I believe this snippet is meant for route change (which I had removed in my earlier PR).  It should just call a valid endpoint. If user is logged out due to session expiry, this will enable the page to be redirected to login page. @wooldridge can you please confirm if that is the case